homeassistant/components/epson/media_player.py

```diff
@@ -176,11 +179,13 @@ async def async_turn_on(self):
         """Turn on epson."""
         if self._state == STATE_OFF:
             await self._projector.send_command(TURN_ON)
+            self._state = STATE_ON
```

Comment:
Why is this needed? An update triggers after changing the state right?

Reply:
After ON projector waits 30 seconds to show give state_on, and 2 minutes to cool down the lamp and after that shows STATE OFF.